salalib/ntfp.h

```diff
@@ -14,43 +15,46 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+#pragma once
 
-
-#ifndef __NTFP_H__
-#define __NTFP_H__
+#include "genlib/p2dpoly.h"
 
 struct NtfPoint {
-   char m_chars;
+   int m_chars;
    int a;
    int b;
-   NtfPoint(char chars = 10)  // apparently 10 is NTF default

```

Comment:
Have you double checked that this class does never get serialized into any file? Changing the variable size can play havoc with serialisation...

Reply:
Yes, this class is only used as a local temporary variable to load NTF files in MetaGraph::loadLineData